test/integration/services/data/test-get-workload-points.js

```diff
@@ -16,7 +16,7 @@ describe('services/data/get-workload-points', function () {
       })
   })
 
-  xit('retrieves the latest points configuration', function (done) {
+  it('retrieves the latest points configuration', function (done) {
```

Comment:
Was the test data changed to resolve this?

Reply:
Not sure what you mean.  Was this test commented out for a reason?  It passes locally